model/src/test/kotlin/config/RepositoryConfigurationTest.kt

```diff
@@ -121,6 +123,18 @@ class RepositoryConfigurationTest : WordSpec({
                     - given: MPL-2.0 or EPL-1.0
                       choice: MPL-2.0
                     - choice: MPL-2.0 AND MIT
+                package_snippet_choices:
```

Comment:
Comparing with `license_choices` a few lines above, should be just call this `snippet_choices`?